.github/workflows/release.yml

```diff
@@ -104,6 +135,15 @@ jobs:
         with:
           signing_key_file: ${{ secrets.SIGNING_KEY_FILE }}
           signing_file_path: ${{ env.SIGNING_KEY_FILE_PATH }}
+      - name: Set github user
+        # We set these credentials because we have no way of getting the users (that started the workflow) email
+        # Reference: https://github.com/actions/checkout/issues/13
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+      - name: Pull updated files
```

Comment:
Do we need this pull step for any of these tasks? Unclear what we'd be pulling here, I thought this action runs off of whatever branch it's run from.

Reply:
@sarahkoop the workflow modifies the `build.gradle` to update the version name and code, then commits it and creates a tag. But the publishing is done on a diferent `job`. Each `job` runs on the state in which the branch is when the workflow starts (if a change a file in a `job`, that change is not translated to other `jobs`). That the reason why it pulls, to have the new `versionCode` and `versionName`

Reply:
`Each job runs in a fresh instance of the virtual environment specified by runs-on.` this phrase perhaps explains it better than me lol